api/pkg/s3/api-response.go

```diff
@@ -21,9 +21,11 @@ import (
 	"net/http"
```

Comment:
Can you please point me to the Requirements document for the Copy Object  feature ? 
1. Specifically, is this only for AWS ?
2. if yes, how different is it from this https://docs.aws.amazon.com/AmazonS3/latest/API/API_CopyObject.html

Another question, what is our behavior, during copy, w.r.t the following
SSE
Versioning
CORS
ACL
Storage class

Reply:
@rhsakarpos , Thanks for your reviewing.  The Requirement for the Copy Object is from AWS.  The PR implement partial features.  It include Copy Object between distinct backends and adding new attribute  and checking pre conditions in http header.  The PR dont support SSE, versioning, CORS, ACL, Storage class.

Reply:
Hi @sunfch if the source object already set SSE, Versioning, etc, will those settings be copied, means set to new object too?

Reply:
@sfzeng , thanks for your review,  SSE and Versioning are not include in the PR now